dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -151,6 +152,10 @@ public class ItemServiceImpl extends DSpaceObjectServiceImpl<Item> implements It
     @Autowired(required = true)
     private EntityTypeService entityTypeService;
 
+
+    @Autowired(required = true)
+    protected MetadataValueDAO metadataValueDAO;
```

Comment:
why?

Reply:
Because it is duplicate of `VersioningWithRelationshipsIT.java`

Reply:
That's mistake, thank you